src/systems/ec/ec-component/ec-component-navigation/ec-component-inpage-navigation/ec-component-inpage-navigation.twig

```diff
@@ -7,41 +7,4 @@
     }
 #}
 
-{# Internal properties #}
-
-{% set _cssClass = 'ecl-inpage-navigation' %}
-{% set _extraAttributes = '' %}
-
-{# Internal logic - Process properties #}
-
-{% if extraClass is defined %}
-  {% set _cssClass = _cssClass ~ ' ' ~ extraClass %}
-{% endif %}
-
-{% if extraAttributes is defined %}
-  {% for attr in extraAttributes %}
-    {% set _extraAttributes = _extraAttributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~'"' %}
-  {% endfor %}
-{% endif %}
-
-{# Print the result  #}
-
-<nav class="{{ _cssClass }}" {{ _extraAttributes }}>
-  <div class="ecl-inpage-navigation__title">{{ title }}</div>
-
-  <div class="ecl-inpage-navigation__body">
-    <div class="ecl-inpage-navigation__trigger" aria-controls="ecl-inpage-navigation-list" aria-expanded="false" id="ecl-inpage-navigation-trigger">&nbsp;</div>
-
-    <ul class="ecl-inpage-navigation__list" aria-hidden="true" aria-labelledby="ecl-inpage-navigation-trigger" id="ecl-inpage-navigation-list">
-      {% for link in links %}
-      <li class="ecl-inpage-navigation__item">
-        {% include '@ecl/ec-component-link' with {
-          'extra_classes': 'ecl-inpage-navigation__link',
-          'variant': 'standalone',
-          }|merge(link|default({}))
-        %}
-      </li>
-      {% endfor %}
-    </ul>
-  </div>
-</nav>
+{% include '@ecl/generic-component-inpage-navigation' with { system: 'ec' } %}
```

Comment:
I understand the need to pass the system to parent template, but this is somehow redundant (we are using an EC component, so we should use system: ec in any cases).
Maybe we could find a way to pass this parameter automatically (could be done in another task, not a blocker)

Reply:
This was the simplest solution I came up to. If you have a better idea, I'll welcome it :)

Reply:
not yet, I'll open a ticket to think about it